public/data/merouanezouaid.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Merouane Zouaid",
+    "bio": "Hello, I am Merouane Zouaid, a sophomore majoring in Computer Engineering at the Higher School of Technology of Agadir. Very passionate about Software Engineering, Web Development, Artificial Intelligence, and Obsessed with Open Source :) .",
+    "avatar": "https://github.com/merouanezouaid.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/meroaunezouaid",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/marwanzouaid1",
+        "icon": "twitter"
+      },
+      {
+        "name": "Contact me",
+        "url": "mailto:marwanzouaid@gmail.com",
+        "icon": "envelope"
+      }
+    ]
+  }
```

Comment:
```suggestion
}
```

Reply:
Should I make a new PR just for the bio add?